.github/workflows/test-disable-cache.yaml

```diff
@@ -36,4 +36,3 @@ jobs:
           rm greading
           openstack object save default greading
           grep -Fxq hello greading
-      
```

Comment:
Missing new line?

Reply:
There's a new line in this two files. The change here is because the removal of some empty whitespaces in the end of file.